ZelBack/src/routes.js

```diff
@@ -636,7 +636,7 @@ module.exports = (app, expressWs) => {
     appsService.appendRestoreTask(req, res);
   });
 
-  app.post('/ioutils/fileupload/:fullpath?/:filename?/:appname?', (req, res) => {
```

Comment:
is this compatible with older versions?

Reply:
no api was changed

Reply:
But that means its not compatible with older versions of home ui.

Reply:
older version using full path as arg now all is done on backend so it is not compatible